### PR TITLE
[SM-389] Event log for service account

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Adds event for when a service account retrieve a secret. Clicking on the ID sends the user to the secrets page with the search field pre-populated with the ID.

## Code changes

- **apps/web/src/app/core/event.service.ts:** Added `Secret_Retrieved` event. and marked events with a service account id as coming from the SDK.
- **apps/web/src/app/organizations/manage/events.component.ts**: Added logic for assigning correct name for service account generated events.
- **bitwarden_license/bit-web/src/app/secrets-manager/secrets/secrets.component.ts**: Wired up search
- **libs/common/src/enums/eventType.ts**: Added event type.

## Screenshots

image

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by Thomas-Avery — COMMENTED** [review]
Looks good. There are a few CI errors and a merge conflict that needs to be resolved.

**Comment by Hinton** [comment]
The CodeScene error is fine, we're not going to refactor that method right now 😭

**Review by Thomas-Avery — APPROVED** [review]

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -6490,16 +6503,13 @@
...
     "message": "Password found in a data breach. Use a unique password to protect your account. Are you sure you want to use an exposed password?"
-  }, 
+  },
   "weakAndExposedMasterPassword": {
     "message": "Weak and Exposed Master Password"
   },
   "weakAndBreachedMasterPasswordDesc": {
     "message": "Weak password identified and found in a data breach. Use a strong and unique password to protect your account. Are you sure you want to use this password?"
   },
-  "important": {
```
Comment (Hinton) [inline apps/web/src/locales/en/messages.json:6500]: Duplicated for some reason.

**Review by Thomas-Avery — APPROVED** [review]
LGTM

**Pull request merged**